docs/CHANGELOG.asciidoc

```diff
@@ -73,6 +74,46 @@ This section summarizes the changes in each release.
 //=== Known Issues
 ////
 
+
+[[release-notes-6.3.2]]
+== {kib} 6.3.2
+coming[6.3.2]
+
+[float]
+=== Bug Fixes
+Discover::
+* [Fixes #19445] Removes outdated Aria attribute {pull}20532[#20532]
+
+Monitoring::
+* Fixes issues with sorting and filtering of {es} nodes {pull}20383[#20383]
++
+This issue fixes the sorting and filtering of rows in the table of {es} nodes 
+in the {kib} Monitoring UI.
+
+Platform::
+* Defaults to running the {es} functional test server from a snapshot locally 
+in the {xpack} directory {pull}20585[#20585]
```

Comment:
Unless you've heard otherwise from @archanid I suspect the PR https://github.com/elastic/kibana/pull/20585 should be omitted from this list.  Test-related PRs usually are, I think.

Reply:
I verified with @archanid that this PR should be included.